src/app/shared/modals/redirect-confirmation/redirect-confirmation.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-redirect-confirmation',
+  templateUrl: './redirect-confirmation.component.html',
+  styleUrls: ['./redirect-confirmation.component.scss']
```

Comment:
empty file ..?

Reply:
Yes this is a simple pop-up and these files are automatically generated by the CLI